service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorService.java

```diff
@@ -0,0 +1,11 @@
+package com.ge.predix.acs.attribute.connector.management;
+
+import com.ge.predix.acs.rest.AttributeConnector;
+
+public interface AttributeConnectorService {
```

Comment:
minor: not sure if it's worth having a interface for this. Given its not a user visible, class and we only have one implementation for it. We could add a interface when we need a 2nd implementation